src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/PopulateUpdateSubmission.java

```diff
@@ -0,0 +1,156 @@
+
+    /**
+     * Method populates values into UpdateSubmission for JSON output
+     */
+    public void populate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+            UpdateSubmission updateSubmission) {
+        updateSubmission.setUserSubmission(overseasEntitySubmissionDto);
+        populateDueDiligence(overseasEntitySubmissionDto, updateSubmission);
+        populatePresenter(overseasEntitySubmissionDto, updateSubmission);
+        populateStatements(overseasEntitySubmissionDto, updateSubmission);
+        populateFilingForDate(overseasEntitySubmissionDto, updateSubmission);
+    }
+
+    private void populateDueDiligence(
+            OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+            UpdateSubmission updateSubmission) {
+        var dueDiligenceDto = Optional.ofNullable(overseasEntitySubmissionDto.getDueDiligence());
+
+        if (dueDiligenceDto.isPresent()) {
+            populateDueDiligenceFiledByAgent(dueDiligenceDto.get(), updateSubmission);
+        } else {
+            Optional.ofNullable(overseasEntitySubmissionDto.getOverseasEntityDueDiligence())
+                    .ifPresent(entityDueDiligenceDto -> populateDueDiligenceFiledByOverseasEntity(
+                            entityDueDiligenceDto, updateSubmission));
+        }
+    }
+
+    private void populateDueDiligenceFiledByOverseasEntity(
+            OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto,
+            UpdateSubmission updateSubmission) {
+        var submissionDueDiligence = new DueDiligence();
+
+        populateCommonDueDiligenceDetails(submissionDueDiligence,
+                overseasEntityDueDiligenceDto.getIdentityDate(),
+                overseasEntityDueDiligenceDto.getName(),
+                overseasEntityDueDiligenceDto.getAddress(),
+                overseasEntityDueDiligenceDto.getSupervisoryName(),
+                overseasEntityDueDiligenceDto.getPartnerName(),
+                overseasEntityDueDiligenceDto.getEmail());
+
+        Optional.ofNullable(overseasEntityDueDiligenceDto.getAmlNumber())
+                .ifPresent(submissionDueDiligence::setAmlRegistrationNumber);
+
+        Optional.of(submissionDueDiligence).ifPresent(updateSubmission::setDueDiligence);
+    }
+
+    private void populateDueDiligenceFiledByAgent(DueDiligenceDto dueDiligenceDto,
+            UpdateSubmission updateSubmission) {
+        var submissionDueDiligence = new DueDiligence();
+
+        populateCommonDueDiligenceDetails(submissionDueDiligence, dueDiligenceDto.getIdentityDate(),
+                dueDiligenceDto.getName(), dueDiligenceDto.getAddress(),
+                dueDiligenceDto.getSupervisoryName(), dueDiligenceDto.getPartnerName(),
+                dueDiligenceDto.getEmail());
+
+        Optional.ofNullable(dueDiligenceDto.getAgentCode())
+                .ifPresent(submissionDueDiligence::setAgentAssuranceCode);
+        Optional.ofNullable(dueDiligenceDto.getDiligence())
```

Comment:
as we learned this morning this field is actually not used any more but as still in model we should keep passing it on.